build.sbt

```diff
@@ -86,20 +86,42 @@ headerSources in Compile ++= {
 }
 
 lazy val root = (project in file("."))
-  .aggregate(`proxy-core`,
-             `proxy-cassandra`,
-             `proxy-postgres`,
-             `java-support`,
-             `java-shopping-cart`,
-             `akka-client`,
-             operator,
-             `tck`,
-             docs)
+  .aggregate(
+    `proxy-core`,
+    `proxy-cassandra`,
+    `proxy-postgres`,
+    `java-support`,
+    `java-shopping-cart`,
+    `akka-client`,
+    `operator`,
+    `tck`,
+    `docs`
+  )
```

Comment:
Should this include `protocols,`?

Reply:
@ignasi35 Not yet, that's: https://github.com/cloudstateio/cloudstate/issues/118